### DISCUSSION TITLE
login page missing on 3.9.3-management

### DISCUSSION THREAD

**Discussion by suhigh** [docker-library/rabbitmq#517] [Q&A] [answered]
on the foot of the page, it shows:"undefined: There is no template at js/tmpl/login.ejs undefined", please check it, thanks!

**Comment by michaelklishin** [comment, marked as answer]
[Clear your browser cache](https://www.rabbitmq.com/upgrade.html#rabbitmq-plugins-compatibility) as upgrade docs suggest.
